src/accordion.test.visuals.ts

```diff
@@ -1,98 +1,85 @@
 import { expect, test } from '@playwright/test';
+import type GlideCoreAccordion from './accordion.js';
 
-test.describe('Accordion', () => {
-  test.describe('Light Mode', () => {
-    test('open', async ({ page }, test) => {
-      await page.goto('/iframe.html?id=accordion--accordion');
-
-      const handle = await page.waitForSelector('glide-core-accordion');
-
-      await handle?.evaluate((node) => {
-        node.open = true;
-        return node.updateComplete;
-      });
-
+test.describe('accordion--accordion', () => {
+  test.describe('globals=theme:light', () => {
+    test('open=${false}', async ({ page }, test) => {
+      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
+      await page.locator('glide-core-accordion').waitFor();
       await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
     });
 
-    test('closed', async ({ page }, test) => {
-      await page.goto('/iframe.html?id=accordion--accordion');
-      await page.waitForSelector('glide-core-accordion');
+    test('open=${true}', async ({ page }, test) => {
+      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
+
+      await page
+        .locator('glide-core-accordion')
+        .evaluate<void, GlideCoreAccordion>((element) => {
+          element.open = true;
+        });
+
       await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
     });
   });
 
-  test.describe('Dark Mode', () => {
-    test('open', async ({ page }, test) => {
-      await page.goto(
-        '/iframe.html?id=accordion--accordion&globals=theme:dark',
-      );
-
-      const handle = await page.waitForSelector('glide-core-accordion');
```

Comment:
Playwright frowns on `waitForSelector`. It's slightly less verbose than `page.locator` because it doesn't require type parameters. But I assume it's only a matter of type before it's deprecated. [Many](https://playwright.dev/docs/api/class-elementhandle#element-handle-wait-for-selector) of the methods on the handle it returns are already deprecated.